src/ui/components/search/searchcomponent.js

```diff
@@ -759,4 +755,24 @@ export default class SearchComponent extends Component {
   focusInputElement () {
     DOM.query(this._container, this._inputEl).focus();
   }
+
+  /**
+   * Updates the Search inputs clear button based on the current input value
+   *
+   * @param {Element} button
+   * @param {string} input
+   */
+  _updateClearButton (button, input) {
```

Comment:
would it make sense to add a this.getClearButton() instead of passing in the button? It looks like we have DOM.query(this._container, '.js-yxt-SearchBar-clear'); in two different places already so maybe we could consolidate those?